hasher-matcher-actioner/hmalib/common/tests/test_action_rule_evaluation.py

```diff
@@ -144,3 +145,33 @@ def test_get_action_labels(self):
             action_label_to_action_rules,
             "enqueue_sailboat_for_review_action_label should be in action_label_to_action_rules",
         )
+
+    def test_webhook_action_repalcement(self):
+        content_id = "cid1"
+        content_hash = "0374f1g34f12g34f8"
+
+        banked_signal = BankedSignal(
+            banked_content_id="4169895076385542",
+            bank_id="303636684709969",
+            bank_source="te",
+        )
+
+        match_message = MatchMessage(
+            content_id,
+            content_hash,
+            [banked_signal],
+        )
+
+        action_performers = [
+            performer_class(
+                name="EnqueueForReview",
+                url="https://webhook.site/d0dbb19d-2a6f-40be-ad4d-fa9c8b34c8df",
```

Comment:
not blocking: Can you just use something like http://httpstat.us/404 instead to make it clear that this link won't stick around? feels a bit weird to have the test make an external request at all tbh

Reply:
I want to use a real url for the same reason I included a real URL in the `if __name__ == "__main__":`. This is more of an integration test than a unit test. We should check that the correct url shows up at https://webhook.site/d0dbb19d-2a6f-40be-ad4d-fa9c8b34c8df.

Also, fwiw, this url shows up in other test cases already and is non permanent so after a few days it will not be visible

Going to merge since you marked as non blocking